app/config.js

```diff
@@ -20,6 +20,10 @@ config.session = {
   resave: true,
   saveUninitialized: true,
   secret: process.env.COOKIE_SECRET || 'shh-its-a-secret',
+  cookie: {
+    // In milliseconds. Defaults to 8 hours
+    maxAge: (Number.parseInt(process.env.COOKIE_MAXAGE, 10) || 28800) * 1000,
```

Comment:
I think if `process.env.COOKIE_MAXAGE` is undefined, this will error rather than defaulting as `parseInt()` will choke.

Instead maybe do something like `cookieSeconds: process.env.COOKIE_MAXAGE || 28800` and then multiply * 1000 after.

I think.

Reply:
Actually, maybe not.

Reply:
Nope. Ignore. All is well.

:disappear:

Reply:
No worries. I tested from the console, etc... and it seems like `Number.parseInt()` is coping well with whatever you throw at it (as in it would return `NaN` when it can't convert the thing to an integer).

As `NaN` is falsey (assuming that is a word) it would then fall back to `|| 28800`, etc...

No worries. Thanks for having a look Clive :)